app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/screens/restaurant/nearby/NearbyRestaurantItem.kt

```diff
@@ -24,10 +25,13 @@ import java.util.*
 
 @Composable
 fun NearbyRestaurantItem(
-    restaurant: Restaurant
+    restaurant: Restaurant,
+    onItemClicked: (restaurant: Restaurant) -> Unit
```

Comment:
```suggestion
    onItemClicked: (Restaurant) -> Unit
```

Not so sure why we need to declare a variable `restaurant` here 🤔 I think we can remove it and just declare the type 😄

Reply:
@thiennguyen0196 Yes, you're right. I added it accidentally.